tests/data/tag_windows.py

```diff
@@ -162,12 +248,194 @@ def testApplicationExecution(self):
         userassist.UserAssistWindowsRegistryEventData,
         attribute_values_per_name, ['application_execution'])
 
+    # Test: data_type is 'windows:registry:key_value' AND
+    #       key_path contains '\\Compatibility Assistant\\Store'
+    event = events.EventObject()
+    event.timestamp = self._TEST_TIMESTAMP
+    event.timestamp_desc = definitions.TIME_DESCRIPTION_UNKNOWN
+
+    event_data = {}
```

Comment:
Don't do it like this. Use the corresponding event data object instead.

The intent of these test are to catch issues when the event data objects change.